tests/test_config.py

```
@@ -0,0 +1,44 @@
+            "mkvmerge": {"path": "mkvmerge"},
+            "multiprocessing": {"thread_count": "auto"},
+        }
+    )
+
+    assert config.task.output_suffix == "_Plex"
+    assert config.task.delete_original_video is True
+    assert config.subtitle.profiles[0].id == "jp_sc"
+
+
+def test_thread_count_validation_reports_field():
+    with pytest.raises(ConfigError, match="concurrency.thread_count"):
```

Ruff (0.15.18)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)